### ISSUE TITLE
Normalize page heading hierarchies: 3

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#230]
Ensure all pages have single hierarchy of headings from h1 on down with no skipped or compressed levels.

Traceability: [Microassist Report 1 > Homepage Template > Headings](https://docs.google.com/document/d/10cfUFx-o_kSGBFKCe4MtL45cyc-c-Gwaz3Dxlrp8Qgg/edit#bookmark=id.o5sardkl3ij1)

**Comment by k2soule** [comment]
Some suggestions from the Accessibility Audit to address heading hierarchies:
"The best practice is to have a single, consistently placed heading level one preferably in the main content area. The home page is currently using the site name/logo as the heading level one, which is a recommended practice."
[However, ] "The site name/logo is used as an h1 for every page; this should only be utilized for the homepage as more topic related h1’s should be used for other pages."
"Heading level 2 can be used for Slider Titles & Footer Headings (Gallery Hours | Library Hours | Support ISAW)...Research article titles...visually distinct section headings..."
"If, for style purposes, ISAW would chooses to have a hidden h1, a possible solution is to create a h1 as the first object in the main area and visibly hide it using CSS
.hidden {
	position: absolute;
	left: -10000px;
	top: auto;
	width: 1px;
	height: 1px;
	overflow: hidden;
	}
"
...and ensure that the heading structure for Mobile is the same as for Desktop.

**Comment by paregorios** [comment]
<img width="401" alt="screen shot 2018-05-24 at 9 45 05 pm" src="https://user-images.githubusercontent.com/263285/40510087-da744176-5f9b-11e8-89bc-f0b134e1b7bc.png">

**Comment by skleinfeldt** [comment]
@paregorios my notes say "skip for now"...

**Comment by paregorios** [comment]
@skleinfeldt yeah this is down to me to figure out where the problems are (e.g., just footer?).

**Comment by paregorios** [comment]
Some aspects of this work have already been done under the auspices of other tickets. Here's what I think we still need to do:

## Iterative headings in listing views should begin with ```<h2>``` instead of ```<h3>```:

 - Event listing (/events)
 - News listing (/news)
 - Mosaic listing (/exhibitions/berthouville/objects)
 - plone default/stock listings (folder etc.)

**Comment by cguardia** [comment]
This is on staging now.

**Comment by paregorios** [comment]
Looks good on production.
